### PR TITLE
Float branch pull request

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Add .gitignore of the Mobile project.
Add README File of the Mobile project.
Add the .hintrc file of the Mobile project.
Updated styling of grid to float in the CSS project file.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -56,280 +54,390 @@ <h1 class="Portfolio-hero-title">I’m Alimony Godfrey Glad<br>to see you!
...
+                  <li>
+                    <textarea
+                      id="message"
+                      name="message"
+                      maxlength="500"
+                      required
+                      class="user_message"
+                      placeholder="Write your message here"
+                    >
+                    </textarea>
```
Comment (mikethreels) [inline portfolio.html:432]: - [ ] due to putting the closing tag for textarea on a new line in the browser it enters a value and the user is able to submit the form without filling in the textarea. also, the placeholder is not visible.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Alimony, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html (hunk 1 above)
Comment (Godfrey-Alimony) [reply portfolio.html]: Done Correction Sir.Thank

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Godfrey-Alimony, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Godfrey-Alimony** [comment]
Nice to meet sir.Thank

**Pull request merged**